### DISCUSSION TITLE
Shelly 2.5 vs Tuya smart switch per tapparelle

### DISCUSSION THREAD

**Discussion by nannifoffi** [indomus/forum#2611] [GENERALE - Discussioni generiche]
Ciao a tutti,

sono un dilettante dell'elettricità e della domotica e vedo che invece qui volate altissimo quindi spero di non aver sbagliato audience e di non annoiarvi coi miei problemi che vi espongo di seguito.
Ho un appartamento con 5 tapparelle comandate con commutatori interbloccati a molla (quelli che quando smetti di premere tornano in posizione). Un paio di anni fa ho deciso di domotizzarli tutti con shelly 2.5. A parte i vantaggi ovvi della domotizzazione ce n'è stato uno che non mi aspettavo e cioè che con il software di shelly ho potuto definire il comportamento dell'interruttore fisico in modalità toggle. Quindi con una semplice pressione la tapparella andava su senza dover stare a premere il pulsante per tutto il tempo. Fin qui tutto bene.
Quest'anno senza sapere il motivo mi si sono bruciati 3 shelly. (i collegamenti sono fatti bene, la terra dell'impianto funziona) a distanza di un mese uno dall'altro. Avevo notato che prima uno aveva iniziato a ronzare. Ho pensato a un collegamento fatto male ma smontato e rimontato era tutto ok ma continuava a ronzare e dopo una settimana ha smesso di funzionare. 
Ho deciso allora (per via del prezzo) di provare su Aliexpress i rele tuya (ce ne sono di mille tipi). L'ho installato e funziona perfettamente ma le impostazione del software tuya sono veramente basiche rispetto a quelle di shelly e non c'è la possibilità di modificare il comprtamento del commutatore fisico. Quindi adesso dal commutatore fisico devo rimanere in posizione per alzare la tapparella. Ho visto che qualcuno parlava di flashare un nuovo firmware sui rele ma qui ci spostiamo in un campo che non mi sento di attraversare. La domanda è: esiste un rele tuya o simile che abbia nel suo modo di lavorare la possibilità di modificare il comprtamento del pulsante fisico? oppure unica soluzione è prendersi 3 shelly per 30 euro l'uno?
Grazie a tutti per quello che saprete dirmi

Nannifoffi

**Comment by indomus** [comment]
Due consigli:
- scegli materiale di qualità (Shelly non è male, così come Sonoff);
- consultati con un elettricista, sempre.
Ti aspettiamo in chat!

**Reply by nannifoffi** [reply]
Grazie mille
sono daccordo sul tema qualità. Infatto avevo preso subito shelly. Il punto è che se mi si bruciano 3 rele l'anno la spesa diventa insostenibile. Magari sono stato sfortunato e ho preso una partita difettata,
Grazie davvero

**Comment by suoko** [comment]
Stesso problema, shelly 2.5 bruciata (prima ha dato problemi, poi ronzava e poi ciao) dopo un paio d'anni. La tuya puoi configurarla per due switch standard come la shelly 2.5?
